services/file_get_info.proto

```diff
@@ -32,75 +42,109 @@ import "query_header.proto";
- * field will be non-empty, the deleted field will be true, its size will be 0, and its contents
- * will be empty.
+ * Query to request file metadata from the Hedera File Service (HFS).<br/>
+ * This query requests all of the information _about_ a file, but none of the
+ * _content_ of a file. A client should submit a `fileGetContents` query to
+ * view the content of a file. File content _may_ also be available from a
+ * block node or mirror node, generally at lower cost.
+ *
+ * File metadata SHALL be available for active files and deleted files.<br/>
+ * File metadata MAY be unavailable for expired files.<br/>
+ * The size of a deleted file SHALL be `0` and the content SHALL be empty.
  */
 message FileGetInfoQuery {
     /**
-     * Standard info sent from client to node, including the signed payment, and what kind of
-     * response is requested (cost, state proof, both, or neither).
+     * Standard information sent with every query operation.<br/>
+     * This includes the signed payment and what kind of response is requested
+     * (cost, state proof, both, or neither).
      */
     QueryHeader header = 1;
 
     /**
-     * The file ID of the file for which information is requested
+     * A file identifier.
+     * <p>
+     * This MUST be the identifier of a file that exists in HFS.<br/>
+     * This value SHALL identify the file to be queried.
      */
     FileID fileID = 2;
 }
 
 /**
- * Response when the client sends the node FileGetInfoQuery
+ * A response to a query for the metadata of a file in the HFS.
  */
 message FileGetInfoResponse {
     /**
-     * Standard response from node to client, including the requested fields: cost, or state proof,
-     * or both, or neither
+     * The standard response information for queries.<br/>
+     * This includes the values requested in the `QueryHeader`
+     * (cost, state proof, both, or neither).
      */
     ResponseHeader header = 1;
 
-
     message FileInfo {
         /**
-         * The file ID of the file for which information is requested
+         * A file identifier.
+         * <p>
+         * This SHALL be the identifier of a file that exists in HFS.<br/>
+         * This value SHALL identify the file that was queried.
          */
         FileID fileID = 1;
 
         /**
-         * Number of bytes in contents
+         * A size, in bytes, for the file.
          */
         int64 size = 2;
 
         /**
-         * The current time at which this account is set to expire
+         * An expiration timestamp.
+         * <p>
+         * The file SHALL NOT expire before the network consensus time
+         * exceeds this value.<br/>
+         * The file SHALL expire after the network consensus time
+         * exceeds this value.<br/>
          */
         Timestamp expirationTime = 3;
 
         /**
-         * True if deleted but not yet expired
+         * A flag indicating this file is deleted.
```

Comment:
```suggestion
         * A flag indicating whether this file is deleted.
```

Reply:
I deliberately removed `whether` from flags.  A set flag indicates a specific condition (deleted), not an alternative.